packages/ember-glimmer/lib/syntax/curly-component.js

```diff
@@ -188,7 +188,6 @@ class CurlyComponentManager {
     aliasIdToElementId(args, props);
 
     props.parentView = parentView;
-    props.renderer = parentView.renderer;
```

Comment:
Tackled this in https://github.com/emberjs/ember.js/pull/14140 (which has caused you yet another rebase conflict).  Sorry...